### ISSUE TITLE
Different Graphs on Tensorboard when running cifar10_train.py from master branch and r0.12 branch

### ISSUE THREAD

**Issue by CTTC** [tensorflow/tensorflow#6079]
I got different views of network graphs when running cifar10_train.py from [master branch](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/models/image/cifar10/cifar10_train.py) and [r0.12 bracnch](https://github.com/tensorflow/tensorflow/blob/r0.12/tensorflow/models/image/cifar10/cifar10_train.py).

So running cifar10_train.py from r0.12 gave a complete graph in which each layer like conv1 and conv2 are connected, just like this:

[cifar10_train.py graph from r0.12 branch][1]

But running cifar10_train.py from master gave several graphs where each layer like conv1 and conv2 are not connected, just like this:

[cifar10_train.py graph from master branch][2]

I saw that cifar10_train.py from master branch used tf.train.MonitoredTrainingSession, is this a bug of this class or is this just what it is expected to generate?

  [1]: https://i.stack.imgur.com/dk0zl.png
  [2]: https://i.stack.imgur.com/NdAFs.png

**Comment by michaelisard** [comment]
You should probably ask this question on (stackoverflow)[http://stackoverflow.com/questions/tagged/tensorflow] since it's a Tensorboard usage question not a bug report.

I am not entirely sure without seeing the running Tensorboard instances, but this may just be a result of the heuristic it uses to decide how to plot things. Notice that in the disconnected case conv1 has an input from shuffle_batch, and conv2 has an input from norm1, just as in the connected case. In the disconnected case if you click around e.g. conv2 is there an 'add to main graph' option? That might result in more what you are expecting.
